client/src/components/Site.vue

```diff
@@ -183,9 +183,11 @@ export default {
         },
         'Weather Barometric Pressure Value': {
           data: [],
-          limit: {
-            default: null
-          }
+          limit:
+            {
+              // leave line below commented so that the pressure plot y axes auto-scales
+              //default: null
```

Comment:
We shouldn't leave commented code in the source. Just remove the default: line and leave a comment explaining why it needs to be an empty object.